### DISCUSSION TITLE
Veritabanı nedir?

### DISCUSSION THREAD

**Discussion by mehmetelbeyli79** [ademilter/homepage#63] [Q&A] [answered]
Merhaba kullandığınız sitenin backend'de kullandığınız veritabanı nedir? Yazılarınız vb. nereye kaydediliyor yada bir api'den mi çekiliyor o da?

**Comment by ademilter** [comment, marked as answer]
readme'de yazdığımı sanıyordum, şimdi ekledim. https://github.com/ademilter/homepage/blob/master/README.md#blog

[contentlayer](https://www.contentlayer.dev/) kullanıyorum
